lua/bufferline/highlight.lua

```diff
@@ -72,8 +73,8 @@ return {
     local fg_current = hl.fg_or_default({'Normal'}, '#efefef', 255)
     local fg_inactive = hl.fg_or_default({'TabLineFill'}, '#888888', 102)
     --- @type barbar.utils.hl.group
-    local fg_target = {cterm = 'red'}
-    fg_target.gui = fg_target.cterm
+    local fg_target = {gui = 'red'}
```

Comment:
Fixing diagnostics. `cterm` can be `integer|string`, but `gui` can _only_ be `string`. So assigning `gui = cterm` here causes an error, since (technically) we could make a mistake later on. However, the reverse is possible.